source/administration/config-settings.rst

```diff
@@ -4714,6 +4714,21 @@ Audit settings
 
 The audit settings output audit records to syslog (local or remote server via TLS) and/or to a local file. Both are disabled by default. They can be enabled simultaneously.
 
+Remote Clusters
```

Comment:
I believe the setting can also be documented in the telemetry doc.

Reply:
Thanks, @amyblais! I've added 'bool RemoteClusters' to the Telemetry doc.